### DISCUSSION TITLE
Inquiry about the release date of the @inertiajs/react package

### DISCUSSION THREAD

**Discussion by iCodePoet** [inertiajs/inertia#1994] [Help]
@reinink, I'm reaching out here since I'm unsure of the best place to ask. Please forgive me if this is not the appropriate forum.

I'm experiencing an issue where the back button doesn't work properly on iOS device browsers. Recently, I noticed that a PR addressing a similar issue has been merged. 
- https://github.com/inertiajs/inertia/pull/1992
- https://github.com/inertiajs/inertia/pull/1984

I would like to apply this fix to the service we are currently running in production as soon as possible. Could you let me know the expected release date for this update?

I am currently using [@inertiajs/react version 1.2.0](https://www.npmjs.com/package/@inertiajs/react?activeTab=versions)

Lastly, I want to express my gratitude for creating this package. Thanks to your work, I am happily able to do my job.

**Comment by reinink** [comment]
Hey hoping to have it out this week!
